### ISSUE TITLE
Missing support of momentum in SGD optimizer

### ISSUE THREAD

**Issue by Leo-LiHao** [SciSharp/TensorFlow.NET#907]
When I use the SGD optimizer, I noticed that it only supports setting the learning rate. Is there a way we can add support for momentum (and decay if possible) in SGD?
